internal/common/sink.go

```diff
@@ -0,0 +1,97 @@
+package common
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+	"path"
+	"strings"
+)
+
+// A SanitizerFn sanitizes the input string
+type SanitizerFn func(string) string
+
+// Sink is a struct that identifies the name of the sink, the associated URL and
+// the HTTP method
```

Comment:
I was wondering here if it was actually accurate to say that a `Sink` is associated with a single "HTTP" method the way it is implied in the doc comment. I think a sink is associated with a name, handler and url and it could be exercised by different HTTP methods. I know that this comment was here before this PR, so this is not an objection, more like asking for your opinion. If you agree we might just want to update the description?

Reply:
You're right, and I actually removed the method from the struct when moving it to this file. I missed the comment though - good catch! 🙂